client/signup/steps/pressable-store/index.jsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import EmailValidator from 'email-validator';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import Gridicon from 'components/gridicon';
+
+import LoggedOutForm from 'components/logged-out-form';
+import LoggedOutFormFooter from 'components/logged-out-form/footer';
+import LoggedOutFormLinks from 'components/logged-out-form/links';
+import LoggedOutFormLinkItem from 'components/logged-out-form/link-item';
+import FormTextInput from 'components/forms/form-text-input';
+import FormInputValidation from 'components/forms/form-input-validation';
+import FormButton from 'components/forms/form-button';
+import FormLabel from 'components/forms/form-label';
+import FormSectionHeading from 'components/forms/form-section-heading';
+
+import renderHeroImage from './hero-image';
+
+export default React.createClass( {
+	displayName: 'PressableStoreStep',
+
+	propTypes: {
+		stepName: React.PropTypes.string.isRequired,
+		goToNextStep: React.PropTypes.func.isRequired,
+		signupDependencies: React.PropTypes.object.isRequired,
+	},
+
+	getInitialState() {
+		return {
+			email: '',
+			valid: false,
+			error: null,
+		};
+	},
+
+	getDefaultProps() {
+		return {};
+	},
+
+	componentDidMount() {
+		if ( this._input ) {
+			this._input.focus();
+		}
+	},
+
+	onEmailChange( event ) {
+		const email = event.target.value;
+		const valid = EmailValidator.validate( email );
+		this.setState( {
+			email,
+			valid,
+		} );
+		if ( this.state.error && valid ) {
+			this.setState( {
+				error: null,
+			} );
+		}
+	},
+
+	onSubmit( event ) {
+		event.preventDefault();
+
+		if ( ! this.state.valid ) {
+			this.setState( {
+				error: this.translate( 'Please provide a valid email address.' ),
+			} );
+			return;
+		}
+
+		window.location.href = `https://my.pressable.com/signup/five-sites?email=${ encodeURIComponent( this.state.email ) }&wp-ecommerce=true`;
+	},
+
+	renderStoreForm() {
+		return (
+			<div>
+				<LoggedOutForm className="pressable-store__form">
+					{ renderHeroImage() }
```

Comment:
From the looks of it, this can be treated as a component if you require it as `HeroImage`, allowing you to replace:

```diff
- { renderHeroImage() }
+ <HeroImage />
```